src/components/pages/lesson.js

```diff
@@ -32,6 +32,7 @@ import {
   removeItemAt,
   replaceItemAt
 } from '../../utils/array-utils.js';
+import FavoriteStar from '../favorite-star.js';
```

Comment:
🙈 my eeeeyes! 😂

Reply:
I'm grief-stricken. How could this happen? Multiple times? Order is restored now.

Reply:
😂 I felt the tectonic plates settling back into place